src/algebra/big_operators.lean

```diff
@@ -646,6 +646,10 @@ lemma sum_range_id (n : ℕ) : (finset.range n).sum (λi, i) = (n * (n - 1)) / 2
 by rw [← sum_range_id_mul_two n, nat.mul_div_cancel]; exact dec_trivial
 
 end gauss_sum
+
+lemma card_eq_sum_ones (s : finset α) : s.card = s.sum (λ _, 1) :=
```

Comment:
I don't know if the rule "simple expression on the right" applies to non-`[simp]`-rules. But if so, this equality needs to be reversed. Also, this should be generalized where `1` lives in an arbitrary semiring (or something).

Reply:
This wasn't actually used in the sensitivity proof, it must have been left over from an old attempt. I wasn't sure if it was worth PRing at all. If it's useful at all, it seems more useful in this direction -- definitely not a simp rule, but for rewriting `s.card` to set up an application of `congr` or something like that. (The rhs seems unlikely to show up naturally, especially with 1 being specifically a nat.)